src/webgpu/api/operation/render_pass/storeOp.spec.ts

```diff
@@ -0,0 +1,34 @@
+export const description = `API Operation Tests for RenderPass StoreOp.
+
+  Test Coverage Needed:
+
+  - Test that a render pass with color attachment storeOp set to 'clear' has correct output for:
+      - All color renderable formats
+      - All color attachments from '0' to 'MAX_COLOR_ATTACHMENTS'
+      - Zero/Non-zero Mip.
+
+  - Test that a render pass with color attachment storeOp set to 'store' has correct output for:
+      - All color renderable formats
+      - All color attachments from '0' to 'MAX_COLOR_ATTACHMENTS'
+      - Zero/Non-zero Mip.

```

Comment:
optional: maybe for readability, could dedup parts of this. Also can use the conventions I'm drafting (linked from test plan I'm drafting and haven't published: https://hackmd.io/@kainino0x/H1MwoqqAU)

```
  - Test that a render pass with color attachment storeOp has correct output for combinations of:
      - storeOp set to {'clear', 'store'}
      - All color renderable formats
      - All color attachments from '0' to 'MAX_COLOR_ATTACHMENTS'
      - Zero/Non-zero Mip.
```

and similar below

Reply:
I already forgot where I wrote this, but yeah, let's test the default values in operation tests. We can just add "undefined" to the list of options to test which should make most default values very easy to test.

Reply:
Added in guidelines: https://hackmd.io/@kainino0x/SJtUks9CU

Reply:
Done.